src/ActionsImporter/Models/Feature.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Text.Json.Serialization;
+
+namespace ActionsImporter.Models;
+
+public class Feature
+{
+    public string Name { get; set; } = string.Empty;
+
+    public string Description { get; set; } = string.Empty;
+    [JsonPropertyName("env_name")]
+
```

Comment:
```suggestion

    [JsonPropertyName("env_name")]
```

nit: csharp convention is to place attributes directly above the property they're set on